docs/BreakingChanges/098 Multiple items in a single TableLayoutPanel cell may be rearranged.md

```diff
@@ -0,0 +1,31 @@
+## 98: Multiple items in a single TableLayoutPanel cell may be rearranged
```

Comment:
{
  "Id": "98",
  "Title": "Multiple items in a single TableLayoutPanel cell may be rearranged",
  "Details": "In the .NET Framework 4.5, if multiple items are placed in the same TableLayoutPanel cell, they may be displayed in a different order than they were in the .NET Framework 4.0.",
  "Suggestion": "This behavior was reverted in a servicing update for the .NET Framework 4.5. Please update the .NET Framework 4.5, or upgrade to .NET Framework 4.5.1 or later, to fix this issue. Alternatively, avoid the ambiguous case of multiple items in the same TableLayourPanel cell.",
  "Link": null,
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Forms.TableLayoutControlCollection.Add(System.Windows.Forms.Control,System.Int32,System.Int32)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems#_a=edit&id=449531&triage=true",
  "Notes": "Although we're looking for TableLayoutPanel APIs, what's really needed here is an analyzer that can look for XAML with multiple items in a single TableLayoutPanel cell\nSource analyzer status: Pri 1",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Windows.Forms.TableLayoutControlCollection.Add(System.Windows.Forms.Control,System.Int32,System.Int32)